src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -56,30 +65,188 @@ public function getLockOwner(): ?string
 
     /**
      * Get the entry funding link
-     *
-     * @psalm-return null|object{url: string, title: string}
      */
     public function getFunding(): ?stdClass
+    {
+        return $this->getPodcastIndexFunding();
+    }
+
+    /**
+     * Get the entry funding link
+     */
+    public function getPodcastIndexFunding(): ?stdClass
     {
         if (array_key_exists('funding', $this->data)) {
-            return $this->data['funding'];
+            /** @var stdClass $object */
+            $object = $this->data['funding'];
+            return $object;
         }
 
         $funding = null;
 
         $nodeList = $this->xpath->query($this->getXpathPrefix() . '/podcast:funding');
 
         if ($nodeList->length > 0) {
+            /** @var DOMElement $item */
+            $item           = $nodeList->item(0);
             $funding        = new stdClass();
-            $funding->url   = $nodeList->item(0)->getAttribute('url');
-            $funding->title = $nodeList->item(0)->nodeValue;
+            $funding->url   = $item->getAttribute('url');
+            $funding->title = $item->nodeValue;
         }
 
         $this->data['funding'] = $funding;
 
         return $this->data['funding'];
     }
 
+    /**
+     * Get the podcast license
+     */
+    public function getPodcastIndexLicense(): ?stdClass
```

Comment:
For all new methods, can we return `object` instead of `stdClass`

```suggestion
    public function getPodcastIndexLicense(): object|null
```